Code.gs

```diff
@@ -310,50 +305,46 @@ function ParseOrganizerName(veventString){
 function ParseNotificationTime(notificationString){
   //https://www.kanzaki.com/docs/ical/duration-t.html
   var reminderTime = 0;
-
+  
   //We will assume all notifications are BEFORE the event
   if (notificationString[0] == "+" || notificationString[0] == "-")
     notificationString = notificationString.substr(1);
-
+  
   notificationString = notificationString.substr(1); //Remove "P" character
-
+  
   var secondMatch = RegExp("\\d+S", "g").exec(notificationString);
   var minuteMatch = RegExp("\\d+M", "g").exec(notificationString);
   var hourMatch = RegExp("\\d+H", "g").exec(notificationString);
   var dayMatch = RegExp("\\d+D", "g").exec(notificationString);
   var weekMatch = RegExp("\\d+W", "g").exec(notificationString);
-
+  
   if (weekMatch != null){
     reminderTime += parseInt(weekMatch[0].slice(0, -1)) & 7 * 24 * 60 * 60; //Remove the "W" off the end
-
+    
     return reminderTime; //Return the notification time in seconds
   }
   else{
     if (secondMatch != null)
       reminderTime += parseInt(secondMatch[0].slice(0, -1)); //Remove the "S" off the end
-
+    
     if (minuteMatch != null)
       reminderTime += parseInt(minuteMatch[0].slice(0, -1)) * 60; //Remove the "M" off the end
-
+    
     if (hourMatch != null)
       reminderTime += parseInt(hourMatch[0].slice(0, -1)) * 60 * 60; //Remove the "H" off the end
-
+    
     if (dayMatch != null)
       reminderTime += parseInt(dayMatch[0].slice(0, -1)) * 24 * 60 * 60; //Remove the "D" off the end
-
+    
     return reminderTime; //Return the notification time in seconds
   }
 }
 
-function sameEvent(x){
-  return x.id == this;
-}
-
 function CheckForUpdate(){
   var alreadyAlerted = PropertiesService.getScriptProperties().getProperty("alertedForNewVersion");
   if (alreadyAlerted == null){
     try{
-      var thisVersion = 3.0;
+      var thisVersion = 2.0;
```

Comment:
I'm assuming this is marked as a change because your branch missed my latest commit https://github.com/derekantrican/GAS-ICS-Sync/commit/e8ab2e363de4d5d1d16295e177ab8727d2da5e8c